src/app/account/auth/signup/page.tsx

```diff
@@ -0,0 +1,148 @@
+"use client";
+
+import { useState, FormEvent } from "react";
+import { AuthCard } from "@/common/components/Auth";
+import { Button } from "@/common/components/Button";
+import { EnvelopeIcon } from "@/common/components/CustomIcon/EnvelopeIcon";
+import { LockIcon } from "@/common/components/CustomIcon/LockIcon";
+import { Input } from "@/common/components/Input";
+import { EyeIcon } from "@/common/components/CustomIcon/EyeIcon";
+import { EyeSlashIcon } from "@/common/components/CustomIcon/EyeSlashIcon";
+
+export default function SignUp() {
+  const [email, setEmail] = useState("");
+  const [pwd, setPwd] = useState("");
+  const [cfmPwd, setCfmPwd] = useState("");
+  const [formSubmittedLoading, setFormSubmittedLoading] = useState(false);
+  const [emailErrorMsg, setEmailErrorMsg] = useState("");
+  const [pwdErrorMsg, setPwdErrorMsg] = useState("");
+  const [cfmPwdErrorMsg, setCfmPwdErrorMsg] = useState("");
+  const [isPwdVisible, setIsPwdVisible] = useState(false);
+  const [isCfmPwdVisible, setIsCfmPwdVisible] = useState(false);

```

Comment:
I think it's generally not best practice to have this many states. Ideally we can use a single state which is an object for the same purpose. 

Also, how about using react-hook-forms? It'll help us handle validations, error messages, and makes it easy to integrate with our existing inputs

Reply:
thanks for pointing this out! I completely missed the use case here

updated at 8d3cbd71a3d2b85c7a247a9fcaac9a8018a9b166..0e2d94cce4693fdad049a68317a159cc10c8607b